crates/proto/src/lib.rs

```diff
@@ -4,5 +4,12 @@ use serde::{Deserialize, Serialize};
 pub struct Rustacean {
     pub name: String,
     pub image: String,
+    pub bio:String,
     pub github_url: String,
+    pub email:String,
+    pub geographical_location: String,
+    pub personal_website:Option<String>,
+    pub mastodon_fediverse:Option<String>, 
+    pub IRC:Option<String>,
+    pub preferred_social_network:Option<String>,
```

Comment:
I think Social Networks could have an structure where we can determinate:

- Provider
- Username

For the ones where username can be easily build by appending/prefixing it to a URL.

For instance on sites like X and Instagram one can use:

https://x.com/EstebanBorai

Theres other social networks where a channel is specified, for instance in the Fediverse you can have any server in the network so you find solutions like Mastodon/Bluesky where the URL may not be deterministic, for instance Hachyderm.io.

These cases could be a "Custom" for now.

The structure could be:

```rust
enum SocialNetwork {
  Instagram {
    username: String,
  }
  X {
    username: String
  }
  Custom {
    url: Url
  }
}
```

Then instead of `preferred_social_network` we could have `social_networks` and use a `HashSet` instead to make sure they are not duplicated on storage.

For the URL we can use this struct: https://docs.rs/url/latest/url/struct.Url.html